internal/info/info.go

```diff
@@ -26,9 +26,25 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/vdaas/vald/internal/errors"
 	"github.com/vdaas/vald/internal/log"
 )
 
+// InformationProvider represents the runtime information provider.
+type InformationProvider interface {
```

Comment:
should we modify the interface name ?

```suggestion
type Into interface {
```

Reply:
do you mean `Info`? 
@vankichi what do you think?

Reply:
@kevindiu IMO:
comparing other internal packages, `Info` is the meaningful as interface name.
but, if this interface does not satisfy the main logic for info package, you should choice below 2 options:
- not use `Info` as the name of interface
- add some function, e.g. `New()`, into the `Info` interface

Reply:
@vankichi thank you for your reply. Then I think `Info` is better naming here. I will change it to `Info` later.